audio-formats.md

```diff
@@ -61,7 +61,7 @@ In theory, you can send 44 kHz audio with a broadband or narrowband model, but t
 #### More information
 {: #samplingRateMore}
 
--   For more information about sampling rates, see [wikipedia.org/wiki/Sampling](https://wikipedia.org/wiki/Sampling){: external}. Select *Sampling (signal processing)*.
+-   For more information about sampling rates, see [Sampling (signal processing)](https://en.wikipedia.org/wiki/Sampling_%28signal_processing%29){: external}.
```

Comment:
Why the `en.` in `en.wikipedia.org`? I seem to remember you removed a bunch of `en.` occurrences yesterday, and none of the other links you change in this PR use `en.`. Yeah, I'm just being me.

Reply:
The `.en` is there because I missed it. :-)  Thanks!

Reply:
All good!